[Victoria-2018] Added the sponsorship details and prospectus

jrstarke opened the pull request:
*Please title your pull request in this format: The event name and year in the title of the PR, along with a description of what is being changed, i.e., `[CHI-2017] Add Bluth Company as a sponsor`*

wsilva committed:
Add organizers details and updated index page

wsilva committed:
fixing organizers details

bridgetkromhout committed:
Merge pull request #3923 from wsilva/add-sp-2018-organizers-details

[SP-2018] Add organizers details and updated index page

bridgetkromhout committed:
Date-labels for sponsor logo changes

bridgetkromhout committed:
Merge pull request #3924 from devopsdays/sponsor-fixes

Date-labels for sponsor logo changes

bridgetkromhout committed:
Adding Charlotte sponsors

bridgetkromhout committed:
Merge pull request #3925 from devopsdays/charlotte-sponsors

[CLT-2018] adding sponsors

bushong1 committed:
[BMORE-2018] Adding new sponsors

bridgetkromhout committed:
Merge pull request #3926 from devopsdaysbaltimore/bmore/add-sponsors2

[BMORE-2018] Adding new sponsors

bridgetkromhout committed:
Sponsor namespace cleanup

bridgetkromhout committed:
Merge pull request #3927 from devopsdays/sponsor-namespace

Sponsor namespace cleanup

Commit:
Removing speakers page until we have images

Commit:
Adding confirmed day 1 talks

Commit:
Adding confirmed day 2 talks

Commit:
Fixing formatting errors

Commit:
Escaping quotes

Commit:
Fixing formatting on talk

phrawzty committed:
Merge pull request #3928 from jessiepuls/master

[DSM-2018] Adding confirmed speakers to the program

mariuszbury committed:
cph-sponsor-update

MikeRosTX committed:
Merge pull request #3929 from mariuszbury/ready/cph-sponsor-update

[CPH-2018] sponsor-update

yvovandoorn committed:
[AMS2017] - Open Registration

bridgetkromhout committed:
Merge pull request #3930 from devopsdays/yvandoorn/LaunchReg

[AMS2017] - Open Registration

mariuszbury committed:
Update nnit.yml

NNIT requested that we change their logo's URL.

MikeRosTX committed:
Merge pull request #3931 from mariuszbury/patch-1

Update nnit.yml

micgo committed:
[SEA-2018] Added intial program with keynotes

MikeRosTX committed:
Merge pull request #3932 from micgo/sea-add-keynotes-speakers

[SEA-2018] Added initial program with keynotes

mattk42 committed:
[DEN-2018] Add Dynatrace as gold sponsor.

MikeRosTX committed:
Merge pull request #3933 from mattk42/2018-den-add-dynatrace

[DEN-2018] Add Dynatrace as gold sponsor.

Commit:
DSM-2018: Remove silver sponsorship

kmugrage committed:
[SEA-2018] added picture for John Allspaw

kmugrage committed:
Merge pull request #3934 from DevOpsDaysDesMoines/remove-silver

[DSM-2018] Remove silver sponsorship

kmugrage committed:
Merge pull request #3935 from kmugrage/seattle_speakers

[SEA-2018] added picture for John Allspaw

jrstarke committed:
Added the sponsorship details and prospectus

adrianmoisey committed:
[Cape Town 2018] Add date and location

bridgetkromhout committed:
Merge pull request #3936 from adrianmoisey/cape_town_dates

[Cape Town 2018] Add date and location

mattstratton committed:
[CHI-2018] Add dates for 2018 Chicago (#3917)

* Add dates for 2018 Chicago

Signed-off-by: Matt Stratton <matt.stratton@gmail.com>

* Add Chicago sponsor page

Signed-off-by: Matt Stratton <matt.stratton@gmail.com>

* Make chicago home page pretty

Signed-off-by: Matt Stratton <matt.stratton@gmail.com>

* Make chicago home page pretty

Signed-off-by: Matt Stratton <matt.stratton@gmail.com>

* Linkify location on home page

Signed-off-by: Matt Stratton <matt.stratton@gmail.com>

* Remove Sasha Rosenbaum from organizers

* Make chicago prospectus open in new tab

skurylo commented on content/events/2018-victoria/sponsor.md:
```diff
@@ -15,52 +33,44 @@ Gold sponsors get a full table and Silver sponsors a shared table where they can
+    <td>Tickets Included</td>
+    <td class="bronze">1</td>
+    <td class="silver">2</td>
+    <td class="gold">4</td>
```
I thought we were just doing two.  I think 4 is too many for this venue size.

jrstarke commented:
Sure, I can make that change tonight.

On Feb 6, 2018 09:15, "Steven Kurylo" <notifications@github.com> wrote:

*@skurylo* commented on this pull request.
------------------------------

In content/events/2018-victoria/sponsor.md
<https://github.com/devopsvictoria/devopsdays-web/pull/1#discussion_r166376589>
:

>    <tr>
-    <th><i>Sponsor FAQ</i></th>
-    <th><center><b>Answers to questions frequently asked by
sponsors&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</center></b></th>
-    <th></th>
+    <td>Tickets Included</td>
+    <td class="bronze">1</td>
+    <td class="silver">2</td>
+    <td class="gold">4</td>

I thought we were just doing two. I think 4 is too many for this venue size.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/devopsvictoria/devopsdays-web/pull/1#pullrequestreview-94428892>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/AACz9sbQBTMkBKQYlNXgWyb6iXQJ09dYks5tSIivgaJpZM4R6jqi>
.

mattstratton committed:
Add Chicago sharing image

mattstratton committed:
Fix date on social posts for Chicago 2018

skurylo commented on content/events/2018-victoria/sponsor.md:
```diff
@@ -15,52 +33,44 @@ Gold sponsors get a full table and Silver sponsors a shared table where they can
-    <th></th>
+    <td>Tickets Included</td>
+    <td class="bronze">1</td>
+    <td class="silver">2</td>
```
And one ticket for silver, per the chat in slack

mattstratton committed:
Fix sharing text (#3937)

kmugrage committed:
Text changes to clarify open spaces vs curated content. (#3938)

* Text changes to clarify open spaces vs curated content.

* Update about.md

souzaxx committed:
Adding new sponsors and speakers

bridgetkromhout committed:
Update hugoserver.sh

We don't need to change this

bridgetkromhout committed:
Update 2018-maringa.yml

You have not created your program file, so you do not want to link to it because the link will not work.

Here is how you can create it: https://github.com/devopsdays/devopsdays-web/blob/master/utilities/add_program.sh

bridgetkromhout committed:
Update index.md

Commenting out the program until you create it.

jmaggio committed:
add sponsors

jmaggio committed:
update cyberark link

bridgetkromhout committed:
Correcting Twitter handle

bridgetkromhout committed:
Merge pull request #3941 from devopsdays/twitter-handle-correction

Correcting @ryndaniels's Twitter handle

bridgetkromhout committed:
Merge pull request #3940 from jmaggio/master

[CLT-2018] Adding sponsors & Update sponsor link

bridgetkromhout committed:
Merge pull request #3939 from devopsdaysmaringa/devopsdaymaringa

[MAR-2018] - Adding sponsors and speakers

jrstarke committed:
Updated ticket count, removed chairs from sponsor tables

jrstarke committed:
Updated the Sponsorship prospectus with edits

kmugrage committed:
Merge pull request #3942 from jrstarke/victoria-sponsorship

[Victoria-2018] Added the sponsorship details and prospectus

mattstratton committed:
[CHI-2018] Add paypal buttons and prettify sponsor page for Chicago 2018 (#3943)

* Add paypal buttons and prettify sponsor page for Chicago 2018

* Add chicago special sponsors

kmugrage committed:
[SEA-2018] Added Launch Darkly as gold sponsor

kmugrage committed:
Merge pull request #3945 from kmugrage/2018-seattle-sponsors

[SEA-2018] Added Launch Darkly as gold sponsor

WhatsChrisDoing committed:
Adjust number of gold sponsors

WhatsChrisDoing committed:
Add Hashicorp as silver

MikeRosTX committed:
Merge pull request #3946 from WhatsChrisDoing/toronto-sponsor-update

[TOR-2018] Toronto sponsor update

greenkeeper[bot] committed:
Update require-dir to the latest version 🚀 (#3944)

matiasfrndz committed:
[ZRH-2018] Remove speaker John Doe

matiasfrndz committed:
[ZRH-2018] Change speakers page

matiasfrndz committed:
[ZRH-2018] Update speaker info for Tanya Janca

matiasfrndz committed:
[ZRH-2018] Add speaker info for Bernd Erk

matiasfrndz committed:
[ZRH-2018] Add speaker info for Georgiana Gligor

matiasfrndz committed:
[ZRH-2018] Add speaker info for Rachael Byrne

mattk42 committed:
[DEN-2018] Add Contrast Security sponsorship.

bridgetkromhout committed:
Merge pull request #3947 from mattk42/den-2018-add-contrast_security

[DEN-2018] Add Contrast Security sponsorship.

matiasfrndz committed:
[ZRH-2018] Add speaker info for Paige Bernier

matiasfrndz committed:
[ZRH-2018] Add speaker info for Ramón Medrano Llamas

matiasfrndz committed:
[ZRH-2018] Add speaker info for Basil Brunner

matiasfrndz committed:
[ZRH-2018] Add speaker info for Marcin Niewiadomski

matiasfrndz committed:
[ZRH-2018] Add speaker info for Kris Buytaert

matiasfrndz committed:
[ZRH-2018] Add speaker info for Philipp Krenn

matiasfrndz committed:
[ZRH-2018] Add speaker info for Brian Christner

matiasfrndz committed:
[ZRH-2018] Add speaker info for Torben Höft

matiasfrndz committed:
[ZRH-2018] Remove link to proposal page

MikeRosTX committed:
Merge pull request #3948 from matiasfrndz/zrh-2018-speakers

[ZRH-2018] Add speaker info

ecology-chris committed:
added gold and silver sponsors

MikeRosTX committed:
Merge pull request #3949 from cacorriere/adding_sponsors

[ATL-2018] add mesosphere, launch darkly as gold and sonatype as silver"

jrstarke committed:
[Victoria-2018] Cleaned up Sponsorship Table padding
